WordPress/gutenberg
# Mobile - RichText - Fixes an issue where the font size value won't update

- Gutenberg mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/4257

## Description
This PR fixes an issue where changing the font size value wouldn't be reflected in some cases, for example setting a `Normal` font size and then changing it to `Large` it wouldn't be updated unless something else changes in the RichText component, like updating the color.

Now the current size value is stored in the state of the component so it reflects any changes related to it.

**Note**: The are no entries in the changelog since this feature is still under a development flag.

## How has this been tested?
- Open the app
- Type some text in a Paragraph block
- Open the block settings
- Tap on the Font size option
- Change the font size to `Normal`
- Tap on the Back button
- Open the Font size option again
- Tap on the `Large` option
- **Expect** to see the new font size to change in the component

## Screenshots 

<details>
  <summary>Standard theme (iOS)</summary>

https://user-images.githubusercontent.com/4885740/142448852-d41ad81d-88e7-400a-9a9d-0428cab43898.mov

</details>

<details>
  <summary>Block-based theme (iOS)</summary>

https://user-images.githubusercontent.com/4885740/142448860-fd57db11-74d5-41eb-b297-4a0086280c9b.mov

</details>

<details>
  <summary>Standard theme (Android)</summary>

https://user-images.githubusercontent.com/4885740/142449527-aea80f13-4b87-4d21-b54e-42927139f61b.mp4

</details>

<details>
  <summary>Block-based theme (Android)</summary>

https://user-images.githubusercontent.com/4885740/142449561-ce48e654-8936-4966-8993-a02d507fee28.mov

</details>

## Types of changes
Bug fix

## Checklist:
